docs/academy/exercise5.md

```diff
@@ -203,106 +208,106 @@ Flash APIs require that the length of data to write will be a multiple of 256 by
 
 - flash_aligned_writer.c
 
-<p style="text-align: center;">flash_aligned_writer.c</p>
-``` c
-    #include <assert.h>
-    #include <stddef.h>
-    #include <stdint.h>
-
-    #include <avsystem/commons/avs_utils.h>
-
-    #include "flash_aligned_writer.h"
-
-    void flash_aligned_writer_new(uint8_t *batch_buf,
-                                size_t batch_buf_max_len_bytes,
-                                flash_aligned_writer_cb_t *writer_cb,
-                                flash_aligned_writer_t *out_writer) {
-        assert(batch_buf);
-        assert(batch_buf_max_len_bytes);
-        assert(writer_cb);
-
-        out_writer->batch_buf = batch_buf;
-        out_writer->batch_buf_max_len_bytes = batch_buf_max_len_bytes;
-        out_writer->batch_buf_len_bytes = 0;
-        out_writer->write_offset_bytes = 0;
-        out_writer->writer_cb = writer_cb;
-    }
+    <p style="text-align: center;">flash_aligned_writer.c</p>
+    ``` c
```

Comment:
The whole code is still indented not the way it should be. The "code section" is OK itself, the code inside is indented unnecessarily.